resources/multisite_drupal_standard.make

```diff
@@ -170,6 +170,10 @@ projects[diff][version] = "3.2"
 
 projects[ds][subdir] = "contrib"
 projects[ds][version] = "2.11"
+; Issue #2541786 : Changing layout to "fluid 2 columns stacked" resulted in crash
+; https://www.drupal.org/node/2541786
+; https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-330
+projects[ds][patch][] = https://www.drupal.org/files/issues/2541786-8.patch
```

Comment:
I disagree to solve this by patching 2.11
This patch is already included in 2.12
We should instead upgrade DS to 2.14
This is why It's important to include as much as possible module upgrades in 2.3 :)